src/main/java/org/magicdgs/thaplv/cmd/argumentcollections/HaplotypeModelArgumentCollection.java

```diff
@@ -36,32 +36,33 @@
 import org.broadinstitute.hellbender.exceptions.UserException;
 
 /**
- * Argument collection for haplotype models
+ * Argument collection for haplotype models.
  *
  * @author Daniel Gomez-Sanchez (magicDGS)
  */
 public abstract class HaplotypeModelArgumentCollection implements ArgumentCollectionDefinition {
     private static final long serialVersionUID = 1L;
 
-    private final boolean allowDontCheck;
+    private final boolean allowCheckOnly;
 
-    @Argument(fullName = ThaplvArgumentDefinitions.HAPLOTYPE_MODEL_LONG, shortName = ThaplvArgumentDefinitions.HAPLOTYPE_MODEL_SHORT, doc = "Haplotype model for convert calls.", common = false, optional = true)
+    @Argument(fullName = ThaplvArgumentDefinitions.HAPLOTYPE_MODEL_LONG, shortName = ThaplvArgumentDefinitions.HAPLOTYPE_MODEL_SHORT, doc = "Haplotype model for convert calls. Note that conversion tools does not accept CHECK_ONLY as model.", common = false, optional = true)
```

Comment:
Probably the change here should be reflected in the javadoc documentation of the conversion program group.